src/main/java/org/myrobotlab/service/interfaces/Processor.java

```diff
@@ -0,0 +1,18 @@
+package org.myrobotlab.service.interfaces;
+
+public interface Processor {
```

Comment:
Java doc here explaining what this is used for would be good

Reply:
updated